zipkin-ui/js/component_ui/trace.js

```diff
@@ -443,7 +448,11 @@ export default component(function trace() {
     this.on(document, 'uiZoomInSpans', this.zoomInSpans);
     this.on(document, 'uiZoomOutSpans', this.zoomOutSpans);
 
+    this.on(document, 'uiShowJson', this.showJson);
+
     const self = this;
+    self.raw = data.raw;
```

Comment:
sorry I meant how do you control this via the UI. Ex if the user wants to
view the raw json (api/v1/trace/id?raw), what do they do? Or does raw mean
something else?